docs/CodeOfConduct.md

```diff
@@ -0,0 +1,76 @@
+# Open Enclave SDK Code of Conduct
+
+## Our Pledge
+
+In the interest of fostering an open and welcoming environment, we as
+contributors and maintainers pledge to make participation in our project and
+our community a harassment-free experience for everyone, regardless of age, body
+size, disability, ethnicity, sex characteristics, gender identity and expression,
+level of experience, education, socio-economic status, nationality, personal
+appearance, race, religion, or sexual identity and orientation.
```

Comment:
What about _political affiliation_? Also the organization that some contributor works for?

Reply:
I would leave those as upstream suggestions to the Contributor Covenant. There will always be additional differences that we will have failed to enumerate, and folks should take it in the spirit of creating a "harassment-free experience for everyone".